docs/application/vstools/install.md

```diff
@@ -55,17 +55,17 @@ The Tizen Emulator for Visual Studio has the same requirements as the emulator i
 
 
 <a name="install"></a>
-## Visual Studio Tools for Tizen Installation
+## Visual Studio Tools for Tizen installation
 
-To use the Tizen SDK tools, you must install the Visual Studio Tools for Tizen extension and the baseline SDK.
+To use the Tizen SDK tools, you must install the Visual Studio Tools for Tizen extension and the Baseline SDK.
 
-> **Note**  
-> If Tizen Studio is already installed in your computer, simply [set the Tizen Baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
+> [!NOTE] 
+> If Tizen Studio on already installed in your computer, simply [set the Tizen Baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
```

Comment:
"is" is the proper preposition here.
```suggestion
> If Tizen Studio is already installed in your computer, simply [set the Tizen Baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
```